internal/output/progress.go

```diff
@@ -50,7 +50,11 @@ func (d *Spinner) moveCaretBack() int {
 		prevPos = len(d.frames) - 1
 	}
 	prevFrame := d.frames[prevPos]
-	d.out.Fprint(d.out.Config().ErrWriter, fmt.Sprintf(moveCaretBack, len(prevFrame)))
+	if d.out.Config().ShellName != "cmd" { // cannot use subshell/cmd.Name due to import cycle
```

Comment:
Let's use this instead, minimizing hard coded values:

https://github.com/ActiveState/cli/blob/2f23dcf71b56d5f188119d2e5fc4f4eaa80a6175/internal/subshell/cmd/cmd.go#L33

Reply:
My comment next to this conditional addresses this. I wanted to use the constant, but I cannot due to an import cycle.

Reply:
Ok.. not that big a deal. Let's go with this.